src/main/java/seedu/organizer/logic/commands/ToggleSubtaskCommand.java

```diff
@@ -0,0 +1,122 @@
+package seedu.organizer.logic.commands;
+
+import static seedu.organizer.model.Model.PREDICATE_SHOW_ALL_TASKS;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import seedu.organizer.commons.core.Messages;
+import seedu.organizer.commons.core.index.Index;
+import seedu.organizer.logic.commands.exceptions.CommandException;
+import seedu.organizer.model.subtask.Subtask;
+import seedu.organizer.model.subtask.UniqueSubtaskList;
+import seedu.organizer.model.tag.Tag;
+import seedu.organizer.model.task.DateAdded;
+import seedu.organizer.model.task.DateCompleted;
+import seedu.organizer.model.task.Deadline;
+import seedu.organizer.model.task.Description;
+import seedu.organizer.model.task.Name;
+import seedu.organizer.model.task.Priority;
+import seedu.organizer.model.task.Status;
+import seedu.organizer.model.task.Task;
+import seedu.organizer.model.task.exceptions.DuplicateTaskException;
+import seedu.organizer.model.task.exceptions.TaskNotFoundException;
+import seedu.organizer.model.user.User;
+
+/**
+ * Inverse the value of task status
+ */
+public class ToggleSubtaskCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "toggle-subtask";
+    public static final String COMMAND_ALIAS = "ts";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a subttask to a task. "
+            + "Parameters: TASK_INDEX (must be a positive integer) SUBTASK_INDEX (must be a positive integer)";
+
+    public static final String MESSAGE_EDIT_SUBTASK_SUCCESS = "Toogled Subtask: %1$s";
+
+    public final Index taskIndex;
+    public final Index subtaskIndex;
+
+    private Task taskToEdit;
+    private Task editedTask;
+    private Subtask editedSubtask;
+
+    /**
+     * @param taskIndex index of the task in the filtered task list to edit
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)